scripts/start.mjs

```diff
@@ -0,0 +1,35 @@
+import { spawn, exec } from "child_process";
+console.log("Starting a hardhat node...");
+const hardhat = spawn("yarn contracts hardhat node", { shell: true });
```

Comment:
Make sure to listen for the [process exit](https://nodejs.org/api/child_process.html#event-exit). If this process exits we should probably display a message then terminate the node process.